src/Compilers/Core/Portable/Syntax/SyntaxAnnotation.cs

```diff
@@ -18,7 +18,7 @@ public sealed class SyntaxAnnotation : IObjectWritable, IObjectReadable, IEquata
         /// <summary>
         /// A predefined syntax annotation that indicates whether the syntax element has elastic trivia.
         /// </summary>
-        public static readonly SyntaxAnnotation ElasticAnnotation = new SyntaxAnnotation();
+        public static SyntaxAnnotation ElasticAnnotation { get; } = new SyntaxAnnotation();
```

Comment:
This one seems better be a field. Annotations have identity. User would like to know that ElasticAnnotation is a singleton.